app/javascript/components/pages/UserRecipes.js

```diff
@@ -1,17 +1,52 @@
 import React, { Component } from 'react'
-import { Card, CardTitle, Col, Button } from 'reactstrap'
+import { Card, CardTitle, CardText, Col, Button } from 'reactstrap'
 import { NavLink } from 'react-router-dom'
 
 class UserRecipes extends Component {
-  render(){
+  constructor(props) {
+    super(props)
+    this.state = {
+      userrecipes: []
+    }
+  }
+  componentDidMount() {
+    this.userRecipes()
+  }
+  userRecipes = () => {
+    fetch("/user_recipes")
+      .then(response => response.json())
+      .then(userRecipeArray => this.setState({ userrecipes: userRecipeArray }))
+      .catch(errors => console.log("index errors:", errors))
+  }
+  render() {
+    let { userrecipes } = this.state
     return (
-	<>
-    
-    
 
-	</>
+      <>
+        <h1>Your Saved Recipes</h1>
+        <Col sm="6">
+          {userrecipes.map(userrecipe => {
```

Comment:
What is there are no `userrecipes` or if the request to get user recipes fails? How would the user know what to do next or what message should the user see?